src/Notification/UserUnsuspendedBlueprint.php

```diff
@@ -47,7 +40,6 @@ public function getSubject()
      */
     public function getFromUser()
     {
-        return $this->actor;
     }
```

Comment:
Does a reason exist for `getFromUser` to remain in this class? Is it a requirement of an interface by any chance? If so, it should probably return `null` or be removed entirely.

Same applies for the other Blueprint.

Reply:
yes, it's here https://github.com/flarum/core/blob/eb4b18a979c7406cbf154a107662652d282fe415/src/Notification/Blueprint/BlueprintInterface.php#L22-L27 I left it this way to keep up with `getData` (also returns nothing and yet it's in the class) https://github.com/flarum/suspend/blob/245fc135a233a9a13d48c7bbcd3941fb4c626968/src/Notification/UserUnsuspendedBlueprint.php#L53-L58

Reply:
Thanks for adding those. It seems the interface actually does require null or the return value, but don't have proper return type hinting yet 😞